gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/TokenStateServiceMessages.java

```diff
@@ -33,7 +33,7 @@
   @Message(level = MessageLevel.DEBUG, text = "Revoked token {0}")
   void revokedToken(String tokenDisplayText);
 
-  @Message(level = MessageLevel.DEBUG, text = "Unknown token {0}")
+  @Message(level = MessageLevel.DEBUG, text = "Unknown or revoked token {0}")
```

Comment:
I think we can continue treating them as unknown.

Reply:
Addressed in new update.